### PR TITLE
API-8585: Add auto access rules limits

### PR DESCRIPTION
**Pull request by Adimus11**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-8585-add-info-about-limits-for-access-rules)
- [Jira](https://livechatinc.atlassian.net/browse/XXX-XXX)

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

File: content/management/limits/index.mdx [hunk 1]
```diff
@@ -25,6 +25,7 @@ Based on the <a href="https://www.livechat.com/pricing/" target="_blank">billing
 | Number of banned visitors     | 50      | 200   | 400      | 400        |
 | Tracked visitors              | 400     | 400   | 400      | 400        |
 | Customers history             | 2 mos.  | 1 yr. | 2 yr.    | 4 yr.      |
+| Auto Access Rules             | 0       | 200   | 200      | 200        |
```
Comment (olek-p) [inline content/management/limits/index.mdx:28]: It's actually 200 for all plans ([source](https://livechatinc.atlassian.net/browse/API-8584))

**Review by olek-p — CHANGES_REQUESTED** [review]

File: content/management/limits/index.mdx (hunk 1 above)
Comment (Adimus11) [reply content/management/limits/index.mdx:28]: There is info here provided that we should set `0` for starters: https://livechatinc.atlassian.net/browse/API-8526 `Set the limit to 200 rules per license for all available plans (based on current usage in plans) - and 0 for starters` I think its worth asking Marcin to clarify this

Comment (olek-p) [reply content/management/limits/index.mdx:28]: Mos def! Especially that I already implemented the DB changes according to the subtask description. @mdojwa which is correct?

File: content/management/limits/index.mdx [hunk 2]
```diff
@@ -25,6 +25,7 @@ Based on the <a href="https://www.livechat.com/pricing/" target="_blank">billing
 | Number of banned visitors     | 50      | 200   | 400      | 400        |
 | Tracked visitors              | 400     | 400   | 400      | 400        |
 | Customers history             | 2 mos.  | 1 yr. | 2 yr.    | 4 yr.      |
+| Auto Access rules             | 0       | 200   | 200      | 200        |
```
Comment (ale-kacperczyk) [inline content/management/limits/index.mdx:28]:
```suggestion
| Auto access rules             | 0       | 200   | 200      | 200        |
```

Comment (ale-kacperczyk) [reply content/management/limits/index.mdx:28]: Sorry, I checked again and we don't capitalize it 😅

**Review by ale-kacperczyk — APPROVED** [review]
I added one more commit after double-checking spelling in the rest of our docs.
Content-wise, it's all good 👌🏻

**Review by olek-p — APPROVED** [review]

File: content/management/limits/index.mdx (hunk 1 above)
Comment (olek-p) [reply content/management/limits/index.mdx]: For future reference: it's 0 for starters after all

**Pull request merged**